staging/src/k8s.io/client-go/tools/cache/BUILD

```diff
@@ -22,6 +22,7 @@ go_test(
         "store_test.go",
         "undelta_store_test.go",
     ],
+    features = ["-race"],
```

Comment:
it's kinda unfortunate that we have to disable this on all of the tests, rather than just the one file that needs it.

Reply:
Agreed, but it's not worth the ugliness to do it more specifically.

Reply:
Does this mean we won't be running the test at all anymore?

Reply:
No, I think it just runs without race detection enabled.